### PR TITLE
feat: Added MinimumBreadcrumbLevel

### PR DESCRIPTION
**Pull request by bitsandfoxes**
Resolves https://github.com/getsentry/sentry-unity/issues/1029

Because there is no inherent order to the log type and we're already transforming `Unity LogType` to `Sentry LogType` I added individual checkboxes to opt-out of adding breadcrumbs automatically.
Screenshot 2022-10-24 at 16 16 00

### CODE REVIEW

**Review by vaind — CHANGES_REQUESTED** [review]
Not sure it makes sense to do the partial solution. Would it be much more trouble to implement the real solution as described in #60 - that's a superset of this

File: samples/unity-of-bugs/Assets/Resources/Sentry/SentryOptions.asset [hunk 1]
```diff
@@ -25,6 +25,7 @@ MonoBehaviour:
   <AttachScreenshot>k__BackingField: 1
   <ScreenshotQuality>k__BackingField: 2
   <ScreenshotCompression>k__BackingField: 75
+  <AddLogsAsBreadcrumbs>k__BackingField: 1
```
Comment (vaind) [inline samples/unity-of-bugs/Assets/Resources/Sentry/SentryOptions.asset:28]: Doesn't seem a correct name? Some older version of the file? Or where does this come from?

File: src/Sentry.Unity/Integrations/UnityLogHandlerIntegration.cs [hunk 2]
```diff
@@ -149,6 +152,32 @@ private void OnQuitting()
             _hub?.FlushAsync(_sentryOptions?.ShutdownTimeout ?? TimeSpan.FromSeconds(1)).GetAwaiter().GetResult();
         }
 
+
+        internal static bool PassesMinimumBreadcrumbLevel(LogType minimumLogType, LogType logType)
```
Comment (vaind) [inline src/Sentry.Unity/Integrations/UnityLogHandlerIntegration.cs:156]: Maybe also document the chosen older in the comment to make it easier to read? What is the order you've chosen anyway? From my naive look just from the names I'd expect the following order:

```
    Log,
    Warning,
    Exception,
    Assert,
    Error,
```

or maybe `Exception` & `Assert` swapped, but they're actually orthogonal.

Also, `Error` isn't really defined in Unity docs so it's not clear to me whether it's any worse than `Exception` or not.

Comment (vaind) [reply src/Sentry.Unity/Integrations/UnityLogHandlerIntegration.cs:156]: On a second thought... maybe we should give user a set of checkboxes to pick which they want to handle? So we don't have to come up with arbitrary ordering that's always going to make someone unhappy.

Comment (bitsandfoxes) [reply src/Sentry.Unity/Integrations/UnityLogHandlerIntegration.cs]: Actually, this is a very fair point.

**Comment by bitsandfoxes** [comment]
> Looks great. Sorry for being such a nuisance on the previous two versions

You were absolutely right tho. It's much more usable this way.

**Pull request merged**